GooglePay/src/main/java/com/braintreepayments/api/GooglePayClient.java

```diff
@@ -182,39 +181,38 @@ public void createPaymentAuthRequest(@NonNull final GooglePayRequest request,
         braintreeClient.sendAnalyticsEvent("google-payment.selected");
 
         if (!validateManifest()) {
-            callback.onResult(null, new BraintreeException(
-                    "GooglePayActivity was " + "not found in the " + "Android " +
-                            "manifest, or did not have a theme of R.style.bt_transparent_activity"));
+            callback.onGooglePayPaymentAuthRequest(new GooglePayPaymentAuthRequest.Failure(new BraintreeException(
+                    "GooglePayActivity was not found in the Android " +
+                            "manifest, or did not have a theme of R.style.bt_transparent_activity")));
             braintreeClient.sendAnalyticsEvent("google-payment.failed");
             return;
         }
 
         //noinspection ConstantConditions
         if (request == null) {
-            callback.onResult(null, new BraintreeException(
-                    "Cannot pass null " + "GooglePayRequest to " + "requestPayment"));
+            callback.onGooglePayPaymentAuthRequest(new GooglePayPaymentAuthRequest.Failure(new BraintreeException(
+                    "Cannot pass null GooglePayRequest to requestPayment")));
             braintreeClient.sendAnalyticsEvent("google-payment.failed");
             return;
         }
 
         if (request.getTransactionInfo() == null) {
-            callback.onResult(null, new BraintreeException(
-                    "Cannot pass null " + "TransactionInfo to" + " requestPayment"));
+            callback.onGooglePayPaymentAuthRequest(new GooglePayPaymentAuthRequest.Failure(new BraintreeException(
+                    "Cannot pass null TransactionInfo to requestPayment")));
             braintreeClient.sendAnalyticsEvent("google-payment.failed");
             return;
         }
 
         braintreeClient.getConfiguration((configuration, configError) -> {
-            if (configuration == null) {
-                callback.onResult(null, configError);
+            if (configuration == null && configError != null) {
+                callback.onGooglePayPaymentAuthRequest(new GooglePayPaymentAuthRequest.Failure(configError));
                 return;
             }
 
             if (!configuration.isGooglePayEnabled()) {
-                callback.onResult(null, new BraintreeException(
-                        "Google Pay " +
-                                "is not enabled for your Braintree account," +
-                                " or Google Play Services are not configured correctly."));
+                callback.onGooglePayPaymentAuthRequest(new GooglePayPaymentAuthRequest.Failure(new BraintreeException(
+                        "Google Pay is not enabled for your Braintree account," +
```

Comment:
is there an Android specific reason why this isn't one big string?

Reply:
No that was just weird editor line break - fixed